### DISCUSSION TITLE
Longhorn ... only within cluster or dedicated storage cluster for N other clusters?

### DISCUSSION THREAD

**Discussion by kai-uwe-rommel** [longhorn/longhorn#2602] [Q&A]
Hello, after browsing over the Longhorn docs, I have one question for which I have not found an answer:
Do I need a Longhorn deployment in each and every cluster where I want to use it for PVs?
Or can I install a dedicated Longhorn storage cluster (e.g. on bare metal) and then use its storage ressources across many other clusters, e.g. "remotely"?

**Comment by joshimoo** [comment]
Currently the expectation is to use longhorn inside of the same cluster where it's installed. There are some github issues, to add support for using longhorn storage outside of it's installed cluster via a csi driver.
https://github.com/longhorn/longhorn/issues/2407
